index.html

```diff
@@ -1,8 +1,153 @@
 <!DOCTYPE html>
 <html>
    <head>
+      <link rel="stylesheet" type="text/css" href="styles.css">
+      <title>Animal Behavior</title>
    </head>
+
    <body>
-      <!-- Write your HTML in this file! -->
+      <header>
+         <h3 id = "mainHeading">To <s>All</s> Some of the Pets I've Loved Before</h3>
+     <h4>Cats</h4>
+       <nav><ul><li><a href = #callieholly>Holly &amp; Callie</a> |</li>
+         <li><a href = #nugget>Nugget</a> | </li>
+         <li><a href = #accio>Accio</a> | </li>
+         <li><a href = #kahlua>Kahlua</a> | </li>
+         <li><a href = #pebbles>Pebbles</a></li>
+         </nav></ul>
+
+      <h4>Dogs</h4>
+         <nav><ul>
+         <li><a href = #bud>Bud</a> | </li>
+         <li><a href = #malibu>Malibu</a></li>
+         </ul></nav>
+         </header>
```

Comment:
I like the spacing within this header tag, it makes the code a little more readable vs if it was all squished together.